api/krusty/openapicustomschema_test.go

```diff
@@ -141,6 +143,9 @@ spec:
 
 // Test for issue #2825
 func TestCustomOpenApiFieldBasicUsage(t *testing.T) {
+	if val, ok := os.LookupEnv("OPENAPI_TEST"); !ok || val != trueStr {
```

Comment:
added all these to avoid parallelism when running the tests

Reply:
suggestion: make a helper function instead of repeating the whole thing. Maybe it could take a function argument, and automatically run the openapi reset at the end? i.e. `openAPITest(t, func(t *testing.T){// the test})`

Reply:
I really like that suggestion! Done